index.html

```diff
@@ -1181,7 +1189,7 @@ <h3>Types</h3>
 <a href="#status">credentialStatus</a>&nbsp;object
               </td>
               <td>
-A valid <a>credential</a> status <a>type</a>. For example,<br>
+A valid <a>verifiable credential</a> status <a>type</a>. For example,<br>
```

Comment:
hmm, you can use status w/ just plain credentials... so this is wrong, we meant credential.

Reply:
Fixed.